gamemodes/cinema/gamemode/modules/scoreboard/controls/cl_htmlcontrols.lua

```diff
@@ -23,9 +23,10 @@ function PANEL:Init()
 
     self.BackButton.DoClick = function()
         self.BackButton:SetDisabled(true)
-        self.HTML:HTMLBack()
         self.Cur = self.Cur - 1
         self.Navigating = true
+        self.HTML:OpenURL(self.History[self.Cur])
+        self.HTML._nextUrlPoll = RealTime() + .5
```

Comment:
this can probably poll faster

Reply:
it has to be delayed like this cause the polling cause the pages to go back and forth which completely breaks the history system, making the delay shorter may cause bugs

Reply:
Um, alright, i'll take a look later